src/models/clients.js

```diff
@@ -22,6 +22,9 @@ module.exports = (sequelize, DataTypes) => {
       defaultValue: () => {
         return randomBytes(64).toString('hex');
       }
+    },
+    redirectUrl: {
+      type: DataTypes.STRING(2048)
```

Comment:
`allowNull: false` ? Or a null means we can't login ? :)

Another question: do you think a client would want to configure several redirect urls ? (Facebook permits it)